source/upgrade.txt

```diff
@@ -70,6 +70,9 @@ Version 6.x Breaking Changes
   ``tlsCertificateKeyFile`` connection options when you call the
   ``MongoClient.connect()`` method, not when you create the
   ``MongoClient`` instance.
+- The ``Db.command()`` method does not take any command-specific
+  options. To learn more about the supported options, see the
```

Comment:
I assume that since the options are being **removed**, whereas they were previously just deprecated, the driver will throw an error.

Reply:
I think "take" is an issue related to [Don't Humanize Inanimate Objects](https://www.mongodb.com/docs/meta/style-guide/writing/use-effective-verbs/#don-t-humanize-inanimate-objects) (the style guide examples aren't the clearest) because it implies performing an action that is usually associated to an intention.

I think "accept" would be better. I think "command-specific options" also could be substituted for a more definitive description.

Suggestion:

"The Db.command() method only accepts [describe the options it accepts] options. To learn more about these options, see ..."